Adding `Contribute` section (#114)
ansys/pymapdl-examples: doc/source/contributing/write_examples.rst

How to write a good example?
============================

Here are some tips for writing a good example.

Use ``mapdl.convert_script``
----------------------------

If you want to translate a MAPDL script into a PyMAPDL one, you can first

> run the following commands:: 

Reviewer: ```suggestion
run these commands:: 
```
Get rid of "the following" if a code block immediately follows and a colon introduces it. You won't believe how many unnecessary "followings" occur!

    from ansys.mapdl.core import convert_script
    convert_script(path_MAPDL_script)

.. warning::

>     The ``mapdl.convert_script`` function is still on beta state, hence it is advised you check its output.
>     Please, double-check if there are easier ways to write some commands.

Reviewer: ```suggestion
    The ``mapdl.convert_script`` method is still in the beta state. You should check its output.
    There may be easier ways to write some commands.
```
I'd advise that the "For eample, use...." be part of the warning. Otherwise, it's out of context.

    For example, prefer using ``mapdl.eplot()`` rather than ``mapdl.run("EPLOT")``.

Use Python libraries as much as possible
----------------------------------------

> The aim of PyMAPDL is to make possible the creation of workflows combining
> MAPDL and the Python environment.
> Some MAPDL workflows can be very efficient but mixing it with Python libraries
> can simplify the code and make it more visual with new plots.

Reviewer: ```suggestion
The aim of PyMAPDL is to make it possible to create workflows that combine
MAPDL and the Python environment.
Some MAPDL workflows can be very efficient, but mixing them with Python libraries
can simplify the code and allow you to generate more visually effective plots.
```

> Do not hesitate to have a look at the `Python Package Index <pypi_org_>`_
> to discover new Python libraries.

Reviewer: ```suggestion
Do not hesitate to explore the `Python Package Index <pypi_org_>`_
to discover new Python libraries.
```

Graphics and plots
------------------

The more visual your example is, the better.

> Python facilitates the creation of colorful figures: take advantage

Reviewer: ```suggestion
Python facilitates the creation of sophisticate graphics and plots: take advantage
```

of that.

> Libraries like `matplotlib <matplotlib_org_>`_, `pandas <pandas_org_>`_

Reviewer: Here again is  the use of "library" where most of the PyAnsys doc refers to these as packages. I'm not sure if there is right or wrong way. I'm just noting what I see done more frequently.

How to write a good example?
============================

Here are some tips for writing a good example.

Use ``mapdl.convert_script``
----------------------------

If you want to translate a MAPDL script into a PyMAPDL one, you can first
run the following commands:: 

>     from ansys.mapdl.core import convert_script
> 
>     convert_script(path_MAPDL_script)

Reviewer 2: ```suggestion
    from ansys.mapdl.core import convert_script
    convert_script(path_MAPDL_script)
```

.. warning::

>     The ``mapdl.convert_script`` function is under development and
>     needs to be improved.

Reviewer 2: ```suggestion
    The ``mapdl.convert_script`` function is still on beta state, hence it is advised you check its output.
```

>     Please, double-check if there are easier way to write some commands.

Reviewer 3: ```suggestion
    Please, double-check if there are easier ways to write some commands.
```

    For example, prefer using ``mapdl.eplot()`` rather than ``mapdl.run("EPLOT")``.

Use Python libraries as much as possible
----------------------------------------

> The aim of PyMAPDL is to make it possible to create workflows combining

Reviewer 2: ```suggestion
The aim of PyMAPDL is to make possible to create workflows combining
```

MAPDL and the Python environment.
Some MAPDL workflows can be very efficient but mixing it with Python libraries
can simplify the code and make it more visual with new plots.

> Don't hesitate to have a look at the `Python Package Index <pypi_org_>`_

Reviewer 3: ```suggestion
Do not hesitate to have a look at the `Python Package Index <pypi_org_>`_
```

to discover new Python libraries.

Graphics and plots
------------------

> The more visual your example, the better.

Reviewer 2: ```suggestion
The more visual your example is, the better.
```

Python facilitates the creation of colorful figures: take advantage
of that.

Libraries like `matplotlib <matplotlib_org_>`_, `pandas <pandas_org_>`_

> or `pyvista <pandas_org_>`_ can help you to make nice graphics, tables, or plots .

Reviewer 3: ```suggestion
or `pyvista <pandas_org_>`_ can help you create nice graphics, tables, or plots .
```

run the following commands:: 

    from ansys.mapdl.core import convert_script
    convert_script(path_MAPDL_script)

.. warning::

    The ``mapdl.convert_script`` function is still on beta state, hence it is advised you check its output.

>     Please, double-check if there are easier ways to write some commands.

Reviewer: ```suggestion
    Because the ``mapdl.convert_script`` method is still in beta, you should check its output.
    Always determine if there are easier ways to write any of the commands.
```

>     For example, prefer using ``mapdl.eplot()`` rather than ``mapdl.run("EPLOT")``.

Reviewer: ```suggestion
    For example, use ``mapdl.eplot()`` rather than ``mapdl.run("EPLOT")``.
```
Consider using this format for functions so that they are API reference links: :func:`function<ansys.mapdl.core.function>`
